tf/chunkparser.py

```diff
@@ -310,20 +337,20 @@ def generate_fake_pos(self):
         return (planes, integer, probs, winner)
 
 
-    def v3_record(self, planes, i, probs, winner):
+    def v4_record(self, planes, i, probs, winner):
         pl = []
         for plane in planes:
             pl.append(np.packbits(plane))
         pl = np.array(pl).flatten().tobytes()
         pi = probs.tobytes()
-        return self.v3_struct.pack(VERSION, pi, pl, i[0], i[1], i[2], i[3], i[4], i[5], i[6], winner)
+        return self.v4_struct.pack(V4_VERSION, pi, pl, i[0], i[1], i[2], i[3], i[4], i[5], i[6], winner)
```

Comment:
Shouldn't this have some more fields added to its packing?

Reply:
Tests pass now.